tests/report/dashboard/test_keyboard.py

```
@@ -0,0 +1,55 @@
+
+import siliconcompiler.report.dashboard.cli.keyboard as keyboard_mod
+
+
+# Platform-independent tests
+def test_check_key_none(monkeypatch):
+    # Simulate no key pressed
+    if os.name == 'nt':
+        monkeypatch.setattr(keyboard_mod.msvcrt, 'kbhit', lambda: False)
+        assert keyboard_mod.check_key() is None
+    else:
+        monkeypatch.setattr(keyboard_mod.select, 'select', lambda x, y, z, w=None: ([], [], []))
```

Ruff (0.15.2)

[warning] Unused lambda argument: `x`

(ARG005)